risk-map/yaml/risks.yaml

```
@@ -720,6 +921,30 @@ risks:
           testing the model during training, tuning, and evaluation, and removing or
           labeling sensitive data during sourcing, filtering, and processing before
           it's used for training.
+    mappings:
+      mitre-atlas:
+        - AML.T0024
+        - AML.T0024.000
+        - AML.T0024.001
```

If we use the main mitre-atlas element i don't know if we need to use the sub-elements...


```suggestion
      mitre-atlas:
        - AML.T0024
```